bench/accuracy.ml

```diff
@@ -0,0 +1,6 @@
+let () =
+  let t1 = Sys.time () in
+  while t1 = Sys.time () do
+    ()
+  done;
+  Printf.printf "accuracy of sys time = %f" (Sys.time () -. t1)
```

Comment:
I think you're actually measuring precision here :slightly_smiling_face:

Reply:
Terminology is difficult :) I'm removing this file anyway, as we don't need it anymore with Mtime.